cmd/aws-vpc-cni/main.go

```diff
@@ -384,14 +387,28 @@ func validateEnvVars() bool {
 	}
 
 	prefixDelegationEn := utils.GetBoolAsStringEnvVar(envEnPrefixDelegation, defaultEnPrefixDelegation)
-	warmIPTarget := utils.GetEnv(envWarmIPTarget, "0")
-	warmPrefixTarget := utils.GetEnv(envWarmPrefixTarget, "0")
-	minimumIPTarget := utils.GetEnv(envMinIPTarget, "0")
+	warmIPTarget, err, inputWarmIP := utils.GetIntFromStringEnvVar(envWarmIPTarget, defaultWarmIPTarget)
```

Comment:
Minor nit - `inputWarmIP` -> `WarmIPTargetStr`

Similar change for below sections. Ideally we should also change the func name `GetIntFromStringEnvVar` to `GetEnvVar` (maybe?) since it returns both int and string..we can do that as fast followup if there are lot of places to change.

Reply:
Renamed the `GetIntFromStringEnvVar` to `GetEnvVar`, and also renamed the output var. Thanks